DESCRIPTION

```diff
@@ -1,9 +1,12 @@
 Package: multidy
-Version: 0.1
+Version: 0.1.1
 Date: 2019-03-11
 Title: Explorable Multiverse data analysis and reports in R
 Description: Create Explorable Multiverse Analysis in R. Multiverse analysis is a philosophy of statistical reporting where paper authors report the outcomes of many different statistical analyses in order to show how fragile or robust their findings are.
-Author: Abhraneel Sarma <abhsarma@umich.edu>
+Authors@R: c(
+		person("Abhraneel", "Sarma", role = "aut", email = "abhsarma@umich.edu"),
+        person("Matthew", "Kay", role = c("aut", "cre"), email = "mjskay@umich.edu")    
```

Comment:
You should have the "cre" tag, not me --- or at least, we should have a discussion about who makes the most sense to have it. "cre" is a bit of a misnomer, it really is the tag for whoever should be in the "Maintainer" field, i.e. the contact person that CRAN will poke if there are issues with the package. So the question is, do you want to be the package maintainer when this goes to CRAN? See here: https://cran.r-project.org/doc/manuals/r-release/R-exts.html#The-DESCRIPTION-file